src/libs/actions/Report.js

```diff
@@ -1263,6 +1263,7 @@ function editReportComment(reportID, originalReportAction, textForNewComment) {
                 isEdited: true,
                 html: htmlForNewComment,
                 text: textForNewComment,
+                type: originalReportAction.message[0].type,
```

Comment:
Why is this change in this PR? It seems unrelated.

Reply:
It is not. Messages is an array and we were not sending the type in the optimistic action (neither in the action from the server), so when we replace the message fully now, this property was not set and was causing issues (we would render the comment as `fragment.text` [here](https://github.com/Expensify/App/blob/47a776bf229a5704d7c4a337af3b455884cf72f1/src/pages/home/report/ReportActionItemFragment.js#L170)